standard/docs/en/schema/records_reference.md

```diff
@@ -1,30 +1,29 @@
 # Record Reference
 
-Whereas there can be multiple releases concerning a given contracting process, there should be a single **contracting record** for each ocid, providing a summary of all the available data about this particular contracting process.
+Whereas there can be multiple releases concerning a given contracting process, there should be a single **record** for each ocid at a given time, consolidating all the available releases about the contracting process.
```

Comment:
See earlier comment https://github.com/open-contracting/standard/pull/831/files#r270829819

Reply:
Fixed in 61f5cd6